### ISSUE TITLE
Map Loadplanner line items (in confirmation mail) with right ledger ID's in Moneybird

### ISSUE THREAD

**Issue by welseythomas** [welseythomas/loadsplanner#4]
As an accountant, I need to have the revenues at the right ledgers id so that I have my adiministration and reporting correct. 
 
- Product is meetingspace e.g. “The Attic (35 pax)”, “The Koel Theater (8-40 pax)” (see all here: https://loadsplanner.nl/#/meetingspaces) → MB ledger ID = 405571759355463677 (space booking)
- Product is membership e.g. “Membership one day a week” (see all here: https://loadsplanner.nl/#/workplaces) → MB ledger ID = 436469082393413075 (memberships)
- Extra TV / Beamer → MB ledger ID = 405590109352625859 (rentals)
- Lunch of the Day (Vegan & Meaty) p.p. → MB ledger ID = 405571127608345933 (F&B)
- Extra Whiteboard → MB ledger ID = 405590109352625859 (rentals)
- Flipover filling → MB ledger ID = 405590109352625859 (rentals)
- Privileged day pass (conditions apply) → MB ledger ID = 436469082393413075 (memberships)
- Hosting (per daypart, 3.5 hours) → MB ledger ID = 405590028815697274 (Event staff)
- Bar Coins → MB ledger ID = 405571127608345933 (F&B)
- DCP Testing → MB ledger ID = 405590028815697274 (Event staff)
- Light / sound / tech / cinema operator → MB ledger ID = 405590028815697274 (Event staff)
- Resident Coins → MB ledger ID = 405571127608345933 (F&B)

Note: **Always send invoice!** Fallback = default setting like it is now (so no ledger is fine if the text is not recognized)

**Comment by welseythomas** [comment]
We have a challange here: products can have different prices (because of dynamic pricing, period when there are booked in). Suggestion: can we get the price per line item in Moneybird and have discounts calculated sperately?

**Comment by farhanaviary123** [comment]
@welseythomas

I’d like to know how many customers will be affected by the discount. If it’s a small number (2-3 customers), I can hard-code it. Otherwise, I’ll handle it through an external UI.

Please confirm the number of customers and the frequency of the discount application. Once I have that info, I’ll do some further research and get back to you.

**Comment by welseythomas** [comment]
What do you mean with 'hard code'?

**Comment by farhanaviary123** [comment]
@welseythomas I was reviewing this sheet: https://docs.google.com/spreadsheets/d/18FrfYrUCZ16VE_BRucKTaVfwQsIb4_edxzShd20Z7PA/edit?gid=0#gid=0

It looks like some values are missing, such as % VAT, Ledger name, Ledger ID, tax_id and more.

I'm planning out this task, but I'm a bit confused after noticing the missing values.
![Image](https://github.com/user-attachments/assets/21a1e5a7-6cf7-4ef8-ae1e-9d2cc44be373)

**Comment by welseythomas** [comment]
Yes I can imagine - looks like that we don't know exactly what products to
add here. Put this feature for now on hold - I will get back at it in a few
weeks when this becomes more relevant.

**Comment by farhanaviary123** [comment]
Okay got it.
